.github/workflows/test.yml

```diff
@@ -3,6 +3,9 @@ name: Build
 on:
   pull_request:
     branches: [master]
+    
+  schedule:
```

Comment:
doesn't the script have since master?
That means it won't on any package, no?

Reply:
I don't see it in the file. Am I missing it maybe?

Reply:
it's in the main package.json script:
`"test:ci": "lerna run test:ci --since origin/master",`

Reply:
the reason is when you change a package in a pr, only the relevant tests will run

Reply:
Good catch, thanks. Will update it

Reply:
Also what about the slack notification? I think I'd create a different file for this task..
We also don't need the prettier to run daily